_docs/migrate_module.md

```diff
@@ -7,24 +7,44 @@ summary: Complete directions for migrating a module to Vox Pupuli, including the
 
 You will have someone by your side in this process. The general flow is to…
 
+* Ensure you have been added as a member of the voxpupuli org on github.
+* Create an issue with the [checklist](#checklist) below.
 * Prepare your repo for transfer
+    * Ensure github issues are enabled.
     * If this module was created with PDK delete .sync.yaml.
     * Ensure that the module has a correct `LICENSE` file in the docroot that matches the mentioned license in the `metadata.json`.
 * At this point you can transfer your own repository.
 * Ask an admin to
-    * Ensure github issues are enabled.
     * Verify that all webhooks are disabled.
     * Enable `Automatically delete head branches` in the repository settings.
     * Add the `collaborators` team to the module's `Collaborators & Teams` 'Teams' list with `Write` permissions (e.g. [https://github.com/voxpupuli/puppet-gitlab/settings/collaboration](https://github.com/voxpupuli/puppet-gitlab/settings/collaboration) (that link works only for admins).
     * Update the [access permissions](https://github.com/organizations/voxpupuli/settings/secrets/actions) (that link works only for admins) for forge.puppet.com secrets so releases can be published.
 * Add the module to our [modulesync setup][managed_modules].
 * Execute [modulesync][msync] for this module.
-* Our modulesync will delete a `CONTRIBUTING.md` in the root directory and place one at `.github/CONTRIBUTING.md`. Please enhance [our existing template][template] if the version in the docroot contains useful parts.
+* Our modulesync will delete a `CONTRIBUTING.md` in the root directory and use the global organization version in voxpupuli/.github.  Please enhance [the global version][contrib] if the version in the docroot contains useful parts.
 * [Release][release] the first version under Vox Pupuli.
 * Create a GitHub issue for the [FORGE][forge] project and ask to deprecate the old module (and approve the new one if the old one was approved as well).
-* Do you think the module qualifies to be approved? Wait until it is released, then raise a [GitHub Issue][approve] in the Puppetlabs organisation.
 * Write a very short blog post about the migration([example][example]). Write to our [mailinglist](mailto:voxpupuli@groups.io) about the migration/new blogpost.
 
+## Checklist
+```markdown
+Reference: https://voxpupuli.org/docs/migrate_module/
+
+* [ ] Remove PDK `.sync.yaml` if it exists.
+* [ ] Ensure correct `LICENSE`.
+* [x] Enable GitHub Issues.
```

Comment:
If the source repo didn't have issues enabled, it should be enabled.
```suggestion
* [ ] Ensure GitHub Issues are enabled.
```

I also suspect non-admins don't have the permissions to do so, but you cover this with a step do it before migrating so it may be OK.

Reply:
That's why I left it there but default checked it, I assumed that in almost every case the person transferring it to us would be able to enable issues on their side and add the checklist before migration.